packages/sveltekit/README.md

```diff
@@ -222,37 +222,26 @@ For [row level security](https://supabase.com/docs/learn/auth-deep-dive/auth-row
 ```ts
 // src/routes/profile.ts
 import { supabaseServerClient, withApiAuth } from '@supabase/auth-helpers-sveltekit';
```

Comment:
You could add the `User` type to the imports to have it available in the `GetOutput` interface. Like `import { supabaseServerClient, withApiAuth, type User } from "@supabase/auth-helpers-sveltekit";`

We will need the `User` type further down.